collaborative/IOU/heat_index.ipynb

```diff
@@ -480,71 +756,115 @@
    "execution_count": null,
-   "id": "41d76861-93b3-41f7-ba3c-22c772813745",
+   "id": "698f025f-6d81-43a8-9bd5-98dbda9e65a3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "num_heatidx_histmonths = (heatidx_hist_day >= hi_threshold).resample(time='1M').sum()\n",
+    "num_heatidx_histmonths.name = 'Days per month above Heat Index threshold of {}°F'.format(hi_threshold)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d0b730bc-cbd8-45b8-9e59-ea1575298550",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "num_heatidx_projdays.hvplot.line(x='year', by='simulation') *\\\n",
-    "proj_trend.hvplot.line(x='year', color='black', line_dash='dashed', label='trendline')"
+    "num_heatidx_histmonths.hvplot.line(x='time', by='simulation')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "83433596-3d67-4a68-8fb8-909e6e74e363",
+   "id": "3889ae2b-557f-46a4-853a-06ee96901981",
    "metadata": {},
    "source": [
-    "## Step 3: Export\n",
-    "Export any variable of interest here for your needs. Here we've selected the daily historical Heat Index values, but you can pass any variable of interest to `var_to_save` here to customize the data to export. "
+    "### 4d) Export summer counts of days per year\n",
+    "\n",
+    "Like steps 2c and 3c, we'll calculate the min, max, mean, and median trends and format a table for easy export. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "09b0a46e-0702-4657-9159-c1e7d0350dd2",
+   "id": "a3e08238-91cb-403b-b23f-ed2df3dd3060",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "var_to_save = heatidx_hist_day\n",
-    "df_export = var_to_save.drop('Lambert_Conformal').to_dataframe()\n",
-    "df_export"
+    "# calculate summary stats\n",
+    "num_heatidx_histdays_stats = compute_multimodel_stats(num_heatidx_histdays_stats)\n",
+    "\n",
+    "# table format with summary stats\n",
+    "df_to_export = summary_table(num_heatidx_histdays_stats)\n",
+    "df_to_export.head(5) # see first 5 rows"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0108f445-710b-405c-b5c2-44c96c3c8edf",
-   "metadata": {
-    "tags": []
-   },
+   "id": "39496d77-9e76-435d-a602-9fbb74cba462",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "fn = 'heat_index_{}'.format(station_name.replace(\" \", \"_\").replace(\"(\", \"\").replace(\")\", \"\"))\n",
-    "df_export.to_csv('{}.csv'.format(fn))"
+    "fn = 'num_heatidx_histdays_{}'.format(station_name.replace(\" \", \"_\").replace(\"(\", \"\").replace(\")\", \"\"))\n",
+    "df_to_export.to_csv('{}.csv'.format(fn))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb801c38-79a9-4c29-bec5-4336584601b2",
+   "metadata": {},
+   "source": [
+    "**Note**: Any of the data variables that we've calculated throughout this notebook can be exported! "
```

Comment:
Maybe include a note on how to do so? Like, "Any of the data variables that we've calculated throughout this notebook can be exported to `.csv`, like we've been doing throughout this notebook! Feel free to use that code to export any other variables of interest."